src/parser/jobs/blm/modules/Procs.js

```diff
@@ -17,9 +17,16 @@ const THUNDER_ACTIONS = [
 	ACTIONS.THUNDER_IV.id,
 ]
 
+const PROC_BUFFS = [
+	STATUSES.THUNDERCLOUD.id,
+	STATUSES.FIRESTARTER.id,
+]
+
 // Yeah they're the same duration now, but it could change...
-const THUNDERCLOUD_DURATION = 18000
-const FIRESTARTER_DURATION = 18000
+const BUFF_DURATIONS = {
```

Comment:
May want to put these in seconds in `src/data/STATUSES/BLM.js` and then just multiply by `1000` here. Seems to be precedent for other modules and avoids magic number BS.